### ISSUE TITLE
Expose amator's `done` option?

### ISSUE THREAD

**Issue by jdgarcia** [stipsan/scroll-into-view-if-needed#156]
I need to do something after the animation is done but don't see a way of doing it with the current options, outside of doing something like setTimeout using the duration time.
I noticed `aminate` function accepts a `done` option that isn't being used by this package.

**Comment by stipsan** [comment]
Yeah I plan on exposing it in some form here: https://github.com/stipsan/scroll-into-view-if-needed/pull/155
